api-reference/50 Common/utils/pdfExporter/exportDataGrid(options).md

```diff
@@ -4,13 +4,223 @@ module: pdf_exporter
+
+        exportGrid() {
+            const doc = new jsPDF();
+            exportDataGridToPdf({
+                jsPDFDocument: doc,
+                component: this.dataGrid.instance
+            }).then(() => {
+                doc.save('Customers.pdf');
+            })
+        }
+    }
+
+
+    <!-- tab: app.module.ts -->
+    import { BrowserModule } from '@angular/platform-browser';
+    import { NgModule } from '@angular/core';
+    import { AppComponent } from './app.component';
+
+    import { DxDataGridModule, DxButtonModule } from 'devextreme-angular';
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxDataGridModule,
+            DxButtonModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+
+##### Vue
+
+    <!-- tab: Installation command -->
+    npm install jspdf jspdf-autotable
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxButton ...
+                @click="exportGrid()"
+            />
+
+            <DxDataGrid ...
+                :ref="dataGridRef">
+                <!-- ... -->
+            </DxDataGrid>
+        </div>
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DxDataGrid from 'devextreme-vue/data-grid';
+    import DxButton from 'devextreme-vue/button';
+    import { jsPDF } from 'jspdf';
+    import 'jspdf-autotable';
+    import { exporDataGrid as exportDataGridToPdf } from 'devextreme/pdf_exporter';
+
+    const dataGridRef = 'dataGrid';
+
+    export default {
+        components: {
+            DxDataGrid,
+            DxButton
+        },
+        data() {
+            return {
+                dataGridRef
+            };
+        },
+        computed: {
+            dataGrid: function() {
+                return this.$refs[dataGridRef].instance;
+            }
+        },
+        methods: {
+            exportGrid() {
+                const doc = new jsPDF();
+                exportDataGridToPdf({
+                    jsPDFDocument: doc,
+                    component: this.dataGrid
+                }).then(() => {
+                    doc.save('Customers.pdf');
+                });
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: Installation command -->
+    npm install jspdf jspdf-autotable
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import DataGrid from 'devextreme-react/data-grid';
+    import Button from 'devextreme-react/button';
+    import { jsPDF } from 'jspdf';
+    import 'jspdf-autotable';
+    import { exporDataGrid as exportDataGridToPdf } from 'devextreme/pdf_exporter';
+
+    const dataGridRef = useRef(null);
```

Comment:
I suggest moving this line to inside of the `App` component because the DataGrid component belongs it. So, it is sensible to keep a reference to the DataGrid inside `App`.